Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,173 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_marginStart="@dimen/grid_2"
+      android:layout_marginEnd="@dimen/grid_2"
+      android:layout_marginHorizontal="@dimen/grid_0_5"
+      android:layout_marginVertical="@dimen/grid_4">
```

Comment:
Can you also remove this `marginVertical`? A vertical margin here will prevent the AppBarLayout from drawing behind the status bar (which we want to do).

Making this change and then adding `android:fitsSystemWindows="true"` to the below AppBarLayout will make it such that the AppBarLayout adds extra top padding to itself to account for the status bar height.